cfgov/unprocessed/js/molecules/Multiselect.js

```diff
@@ -107,7 +107,7 @@ function Multiselect( element ) { // eslint-disable-line max-statements, inline-
     for ( var i = 0, len = list.length; i < len; i++ ) {
       item = list[i];
 
-      // If the value isn't valid kill the script and propt the developer
+      // If the value isn't valid kill the script and prompt the developer.
       if ( !strings.stringValid( item.value ) ) {
         console.log( '\'' + item.value + '\' is not a valid value' );
```

Comment:
Why are these errors showing up again? Shouldn't they be fixed at the source? This relies on the developer actually checking the console, which isn't guaranteed. `console.log` shouldn't be in production code in my opinion. If it should gracefully fail what about throwing an error and handling the error in a `try… catch` block where that error would matter.